tools/installer/full-node.sh

```diff
@@ -223,10 +302,28 @@ EOF
     print_color $GREEN "Systemd service set up and started successfully."
 }
 
+# Function to check if ufw is installed and open port 26656. We need to open the port to keep the network healthy.
+# By default, at least on Debian vultr, this port is not open to the internet.
+configure_ufw() {
+    if command -v ufw &> /dev/null; then
+        print_color $YELLOW "ufw is installed."
+        read -p "Do you want to open port 26656 for p2p communication? (Y/n): " open_port
+        if [[ $open_port =~ ^[Yy] ]]; then
+            ufw allow 26656
```

Comment:
I think we need to check the existing rules and only add one if it's not already present; otherwise, it seems that duplicate rules accumulate. See this [chatGPT convo](https://chatgpt.com/share/674048b9-e32c-8010-91cb-e963f17dc772).